analytics/index.ts

```diff
@@ -4,6 +4,13 @@ const formatEventName = (eventName: string) =>
   eventName.split(' ').join('-').toLocaleLowerCase();
 
 export const FA_EVENTS = {
+  CHAT: {
+    NEW: 'click/chat/new',
+    NAVBAR: 'click/chat/navbar',
+    WIZARD: 'click/chat/wizard',
+    OPEN: (id: string) => `click/chat/${id}`,
+    OPEN_WIZARD: (id: string) => `click/chat/wizard/${id}`,
```

Comment:
Aren't these events colliding with the events at line 44?

I feel we're double-tracking some of the `OPEN` events, but I'm not sure.

[In this file](https://github.com/SwaprHQ/presagio/blob/8f2b38b8abb5cb9f6de9e980926a61ab96994142/app/components/AiChatBase.tsx#L148), we're tracking the chat opening when we click the Wizard icon, but we're also tracking the chat start now when we click the button.

Reply:
Nice catch, I will merge with the other events. Ones should point if the wizard was opened, others should point if the user created a new chat.